internal/host/statemachine.go

```diff
@@ -347,6 +347,9 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th TransitionHandler) stat
 	sm.AddTransitionRule(stateswitch.TransitionRule{
 		TransitionType: TransitionTypeUnbindHost,
 		SourceStates: []stateswitch.State{
+			stateswitch.State(models.HostStatusInstalling),
+			stateswitch.State(models.HostStatusInstallingInProgress),
+			stateswitch.State(models.HostStatusInstallingPendingUserAction),
```

Comment:
Does this also need the other statuses in `HostInstallingStatuses`?

```
	models.HostStatusPreparingForInstallation,
	models.HostStatusPreparingFailed,
	models.HostStatusPreparingSuccessful,
```

Reply:
Oops yes, thank you! Added them!